avae/vis.py

```diff
@@ -180,8 +183,16 @@ def latent_embed_plot_tsne(xs, ys, mode="", epoch=0, writer=None):
     lats = TSNE(
         n_components=2, perplexity=perplexity, random_state=42
     ).fit_transform(xs)
+    print(".............", classes)
+    if classes is None:
```

Comment:
We do this logic a lot in different places, maybe we can make it into a function in a future restructuring? It doesn't have to be in the PR though.